test/resources/microshift-host.resource

```diff
@@ -76,3 +78,25 @@ Is System OSTree
     ${rc}=    Execute Command    stat /run/ostree-booted
     ...    sudo=True    return_stderr=False    return_stdout=False    return_rc=True
     IF    ${rc} == 0    RETURN    ${TRUE}    ELSE    RETURN    ${FALSE}
+
+Get System Boot Id
```

Comment:
Just FYI: we already have this keyword, though it's hidden in `resources/libostree.py`.
Something to tackle in 4.15